### PR TITLE
Sprint 3 release

### PR DESCRIPTION
**Pull request by sethalt**

### CODE REVIEW

File: frontend/src/app/pg-flow-reg-person-friend-family/pg-flow-reg-person-friend-family.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-pg-flow-reg-person-friend-family',
+  templateUrl: './pg-flow-reg-person-friend-family.component.html',
+  styleUrls: ['./pg-flow-reg-person-friend-family.component.scss']
```
Comment (lauraGgit) [inline frontend/src/app/pg-flow-reg-person-friend-family/pg-flow-reg-person-friend-family.component.ts]: for angular is is necessary to have .scss for each page? seems kind of inefficient.

File: frontend/src/app/application-noncommercial-group/application-noncommercial-group.component.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+   'authorizingOfficerName': 'string',
+   'authorizingOfficerTitle': 'string',
+   'eventName': 'string',
+   'applicantInfo': {
+     'name': 'string',
+     'secondaryName': 'string',
+     'firstName': 'string',
+     'lastName': 'string',
+     'dayPhone': {
+       'areaCode': 555,
```
Comment (sethalt) [inline frontend/src/app/application-noncommercial-group/application-noncommercial-group.component.ts]: We created a model object in the current sprint.

File: frontend/src/app/application-noncommercial-group/application-noncommercial-group.component.html [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+            <h3 class="usa-alert-heading">There are errors in your application</h3>
+            <p class="usa-alert-text">Please correct the errors before submitting.</p>
+          </div>
+        </div>
+
+        <label class="usa-input" for="name">Event name</label>
+        <input id="name" name="name" type="text" required [(ngModel)]="application.eventName" #name="ngModel" />
+        <span class="usa-input-error-message" role="alert" [hidden]="name.valid || name.untouched">Event name is required</span>
+
+        <fieldset>
```
Comment (sethalt) [inline frontend/src/app/application-noncommercial-group/application-noncommercial-group.component.html]: Good idea, and will probably be something we'll implement when we start creating the temp-outfitters form.

File: frontend/src/app/pg-flow-reg-person-friend-family/pg-flow-reg-person-friend-family.component.ts (hunk 1 above)
Comment (sethalt) [inline frontend/src/app/pg-flow-reg-person-friend-family/pg-flow-reg-person-friend-family.component.ts]: We removed those in the current sprint.

File: frontend/src/app/app.module.ts [hunk 4]
```diff
@@ -2,19 +2,54 @@ import { BrowserModule } from '@angular/platform-browser';
...
+import { ApplicationNoncommercialGroupComponent } from './application-noncommercial-group/application-noncommercial-group.component';
+import { ApplicationNoncommercialGroupService } from './application-noncommercial-group/application-noncommercial-group-service';
+import { HomeComponent } from './home/home.component';
+import { LoginComponent } from './login/login.component';
+import { PermitApplicationListComponent } from './permit-application-list/permit-application-list.component';
+import { PermitApplicationViewComponent } from './permit-application-view/permit-application-view.component';
+import { PgFlowStep1Component } from './pg-flow-step-1/pg-flow-step-1.component';
+import { PgFlowStep2Component } from './pg-flow-step-2/pg-flow-step-2.component';
+import { PgFlowRegPersonComponent } from './pg-flow-reg-person/pg-flow-reg-person.component';
+import { PgFlowRegPersonFriendFamilyComponent } from './pg-flow-reg-person-friend-family/pg-flow-reg-person-friend-family.component';
```
Comment (sethalt) [inline frontend/src/app/app.module.ts]: At one point we've talked about creating sub-modules for the different bits of functionality. I think we'll need to start implementing those as the site grows.

**Pull request merged**